File: sdkwork-ai-sdk-typescript/src/types/batch-create-request.ts
/** Batch creation request */
export interface BatchCreateRequest {
  /** 端点类型: /v1/chat/completions, /v1/embeddings, /v1/completions */
  endpoint: string;
  /** 元数据 */
  metadata?: Record<string, string>;
  /** 输入文件ID */
  input_file_id: string;
  /** 完成窗口时间(小时) */
  completion_window?: string;
}

File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/BatchCreateRequest.java
package com.sdkwork.ai.model;

public class BatchCreateRequest {
    private String endpoint;
    private Map<String, String> metadata;
    private String inputFileId;
    private String completionWindow;

    public String getEndpoint() {
        return this.endpoint;
    }
    
    public void setEndpoint(String endpoint) {
        this.endpoint = endpoint;
    }

    public Map<String, String> getMetadata() {
        return this.metadata;
    }
    
    public void setMetadata(Map<String, String> metadata) {
        this.metadata = metadata;
    }

    public String getInputFileId() {
        return this.inputFileId;
    }
    
    public void setInputFileId(String inputFileId) {
        this.inputFileId = inputFileId;
    }

    public String getCompletionWindow() {
        return this.completionWindow;
    }
    
    public void setCompletionWindow(String completionWindow) {
        this.completionWindow = completionWindow;
    }
}
